script/dev_deploy_conf_example.json

```diff
@@ -14,6 +14,8 @@
         "enable-metrics-provider": true
     },
     "hstore": {
+        "image": "hstreamdb/hstream:v0.7.1",
+        "persistent-dir": "/data/store",
```

Comment:
If you want the persistence path to be configurable, you should add this configuration item for both zk and hstore, and you need to set the persistence path correctly in the `remove` command

Reply:
hmmmmm i thought it had

https://github.com/hstreamdb/hstream/pull/808/files#diff-ec82c24afc93d1f21cbe94d2e447831953d48fa6c5c5d9964e0c9adf5ad86e16R1180

https://github.com/hstreamdb/hstream/pull/808/files#diff-ec82c24afc93d1f21cbe94d2e447831953d48fa6c5c5d9964e0c9adf5ad86e16R990

Reply:
Oh, I missed it, sorry.

Then you need to set a `persistent-dir` field for the zookeeper configuration as well. Users may want to persist zk data and hstore data in different places. Or you can make `persistent-dir` a global field that allows users to persist zk and hstore data only on this path.

Reply:
consider add some comment in example configuration for what we have and which among them, is optional? since there is no comments functionalities, what about a readme file for it (or along with whole folder contains

Reply:
https://hstream.io/docs/en/latest/deployment/quick-deploy-ssh.html#create-a-configuration-file just update the docs

Reply:
okay